uvicgse/project-virtual-team-2
# Hotfix for issue #127: selected-commit-diff-panel bug

Hotfix for #127 

The panel was placed above the header causing the reshuffle. 

I re-positioned the panel below the header and made it a bit smaller so that it is readable and doesn't swallow any other panels in the body. 

The styling was causing issues as the panel display was not set to "none" when closed.

This is part of a larger issue where instead of hiding and showing the `display` of panels, the app pushes them to the background or foreground by changing their zIndex.

**To test:** 
- Zoom in the graph until the nodes show singular commits
- Click on the node
- the `selected-commit-diff-panel` should appear detailing the diff
- close it
- examine that the app UI was not erroneously changed

> Why this wasn't implemented in the `file-diff-panel` is beyond me
